packages/serverless-contracts/src/contracts/eventBridge/__tests__/fullContractSchema.type.test.ts

```diff
@@ -9,10 +9,11 @@ const eventBridgeContract = new EventBridgeContract({
   payloadSchema: { type: 'object' } as const,
 });
 
-type Check =
-  FullContractSchemaType<typeof eventBridgeContract> extends JSONSchema
-    ? 'pass'
-    : 'fail';
+type Check = FullContractSchemaType<
+  typeof eventBridgeContract
+> extends JSONSchema
+  ? 'pass'
+  : 'fail';
```

Comment:
I'm a little bit confused by this change, it should disappear with a `pnpm format-fix` at the repo root

Reply:
I removed this commit after fixing the pnpm.lock (cf https://github.com/swarmion/swarmion/pull/996#issuecomment-2041456965)